tests/unit/Config.js

```diff
@@ -6,50 +6,51 @@ const Config = require('../../lib/Config.js');
 
 describe('Config', () => {
     beforeEach(() => {
-        Config.reset();
+        this.config = new Config();
```

Comment:
If you instanciate config each time; then it means that you no longer need to reset it. Which may make sens for these specific tests.